lib/rubygems/util/spdx.rb

```diff
@@ -0,0 +1,305 @@
+module Gem
+  class SPDX
+    NONSTANDARD = 'Nonstandard'.freeze
```

Comment:
@mperhap, @tenderlove, @simi, and @bf4, with a little thought since [my last comment on licenses that are nonstandard but noncommercial](https://github.com/rubygems/rubygems/pull/1249#issuecomment-101779498), this is what I landed on. It's short, it's neutral, and it goes right to the heart of the warning: SPDX doesn't provide an identifier for our license.